### ISSUE TITLE
timeout at 30 seconds when receiving a call from Amazon Chime voice connector

### ISSUE THREAD

**Issue by hpasapp** [jambonz/sbc-inbound#100]
I get a timeout at 30 seconds when receiving a call from Amazon Chime voice connector. The call itself functions perfectly well until it gets hung up.  The Chime logs show a sip status 100 and status 200,  but no status 180 (ringing). Is it possible that Chime requires to see a 180 before the 200, and hangs up because of that?

Anything else I should check to diagnose a 30 seconds SIP trunk time-out?

(Edit: the reason for hangup showing in the chime logs is `Reason: SIP;cause=200;text="ACK timeout"`)

Basically in the voice-connector logs I see:

```
T+0 secs: SIP/2.0 100 Trying
T+0 secs: INVITE
T+2 secs: SIP/2.0 200 OK
T+34 secs: SIP/2.0 200 OK
T+34 secs: BYE Reason: SIP;cause=200;text="ACK timeout"
```
So it appears that:
- missing the ACK, that Chime should send I think?
- missing 180 Ringing, that Chime might expect?

**Comment by davehorton** [comment]
Seems like the ACK sent by Chime does not arrive.  Please download the pcap from the jambonz portal for the call and attach

**Comment by hpasapp** [comment]
Ok. Looks like the pcap is empty. I'm guessing I need to enable this somehow/somewhere?

```
$ ls -lh ~/Downloads/callid-24b2f86c-d58e-446e-a996-a5abe2aca40b.pcap
-rw-r--r--@ 1 [redacted]  staff    24B May 26 19:45 [redacted]/callid-24b2f86c-d58e-446e-a996-a5abe2aca40b.pcap

hexdump -C ~/Downloads/callid-24b2f86c-d58e-446e-a996-a5abe2aca40b.pcap
00000000  d4 c3 b2 a1 02 00 04 00  00 00 00 00 00 00 00 00  |................|
00000010  00 00 01 00 01 00 00 00                           |........|
00000018
```

**Comment by davehorton** [comment]
hmm, then you can ssh to the jambonz server and get the sip logs from /var/log/drachtio/drachtio.log after making a test call

**Comment by davehorton** [comment]
I see the problem, Chime is using tcp and drachtio was configured only to use tcp internally.  I fixed that and it should work now

**Comment by davehorton** [comment]
I will separately look into the issue with downloading pcaps

**Comment by hpasapp** [comment]
It does work now yes :) Thank you :)

**Comment by hpasapp** [comment]
Question: where would we set this configuration? (Is it a configuration, like changing 127.0.0.1 to 0.0.0.0? Or is it something in the code?)

**Comment by davehorton** [comment]
its in `/etc/systemd/system/drachtio.service`

**Comment by hpasapp** [comment]
Thanks!

**Comment by davehorton** [comment]
I have applied a recent bug fix to your system that should resolve the issue with downloading pcaps
